driver_timer

```diff
@@ -0,0 +1,471 @@
+#include <FastLED.h>
+
+#define NUM_LEDS 60
+#define LED_PIN 2
+#define DATA_PIN 7
+
+void showStrip() {
```

Comment:
Would add some comments as to what this function does?

Reply:
What do you mean by this?